app/stac_api/serializers.py

```diff
@@ -131,6 +131,33 @@ def update(self, instance, validated_data):
         return instance
 
 
+class ExtentTemporalSerializer(serializers.Serializer):
+    # pylint: disable=abstract-method
+    start_date = serializers.DateTimeField(format='%Y-%m-%dT%H:%M:%SZ')
+    end_date = serializers.DateTimeField(format='%Y-%m-%dT%H:%M:%SZ')
+
+    def to_representation(self, value):
+        temporal_extent = {"interval": [[value.start_date, value.end_date]]}
+        return temporal_extent
+
+
+class ExtentSpatialSerializer(serializers.Serializer):
+    # This field is completely meaningless currently and is only created
+    # for testing (while working on the temporal extent)
+    bbox = serializers.ListField(
+        child=serializers.ListField(child=serializers.FloatField(required=False)), required=False
+    )
+
+    def to_representation(self, value):
+        bbox = {"bbox": [[value.bbox]]}  # probably one pair of brackets too much here?
+        return bbox
+
+
+class ExtentSerializer(serializers.Serializer):
+    spatial = ExtentSpatialSerializer(source="*")
+    temporal = ExtentTemporalSerializer(source="*")
+
```

Comment:
nice! I didn't know you can build serializers from other serializers!

Reply:
It didn't work quite a while ago, when in the first trials start_date and end_date have been implemented in a flat way, too. I came along a description of nested serializers, but must have done something wring. So I switched to the JSONField then. But on the second try, after a short chat with @ltshb (thanks!) retried and it works :smile: